### PR TITLE
intsights-get-alert-activities - None err when missing some alert data

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] Ready

reference: https://github.com/demisto/etc/issues/22185

### CODE REVIEW

File: Integrations/IntSight/IntSight.py [hunk 1]
```diff
@@ -328,45 +330,48 @@ def get_alert_activity():
...
-            'Initiator': demisto.get(activity, 'Initiator'),
-            'CreatedDate': demisto.get(activity, 'CreatedDate'),
-            'UpdateDate': demisto.get(activity, 'UpdateDate'),
-            'RemediationBlocklistUpdate': extract_remediation(
-                demisto.get(activity, 'AdditionalInformation.RemediationBlocklistUpdate')),
-            'AskTheAnalyst': {'Replies': demisto.get(activity, 'AdditionalInformation.AskTheAnalyst.Replies')},
-            'Mail': extract_mail(demisto.get(activity, 'AdditionalInformation.Mail.Replies')),
-            'ReadBy': demisto.get(activity, 'ReadBy')
+    if not response:
+        demisto.results('Alert activities were not found.')
```
Comment (ShahafBenYakir) [inline Integrations/IntSight/IntSight.py:334]: I am afraid this could break BC. What do you think?

Comment (ShahafBenYakir) [inline Integrations/IntSight/IntSight.py:334]: I am afraid this could break BC. What do you think?

Comment (bakatzir) [reply Integrations/IntSight/IntSight.py:334]: if there is no response, there is nothing to iterate on in response, which can reproduce an error as well.

**Comment by bakatzir** [comment]
@ShahafBenYakir We get 401 now :/

**Pull request merged**